README.md

```diff
@@ -1,86 +1,307 @@
+everything went fine.
+
+Run the command in the Terminal.
+
+```bash
+gradle test
 ```
 
-### Set up test database
+If successful the output will be as the following.
 
-The test database is used by the tests and is cleared routinely. We use this
-to gain control over what is in the database when our tests are running and
-to avoid affecting any important data in your main database.
+```bash
+$ gradle test
+:compileJava UP-TO-DATE
+:processResources UP-TO-DATE
+:classes UP-TO-DATE
+:compileTestJava UP-TO-DATE
+:processTestResources UP-TO-DATE
+:testClasses UP-TO-DATE
+> Building 85% > :test > 50 tests completed
+:test
+
+BUILD SUCCESSFUL
+
+Total time: 4 mins 50.919 secs
+```
 
-To set up the test user and database:
+**If the tests start to fail then check the PostgreSQL server is running and the requirements are properly installed by
+ going through the previous steps.**
+ 
+About to be mentioned a couple of examples of what might have gone wrong in the process if the tests start failing.
 
 ```bash
-# Create a user and database
-createuser tombolo_test
-createdb -O tombolo_test tombolo_test -E UTF8
-psql -d tombolo_test -c "CREATE EXTENSION postgis;"
-psql -d tombolo_test -c "SET NAMES 'UTF8';"
+uk.org.tombolo.core.AttributeTest > testUniqueLabel FAILED
+    java.util.ServiceConfigurationError
+        Caused by: org.hibernate.service.spi.ServiceException
+            Caused by: org.hibernate.exception.JDBCConnectionException
+                Caused by: org.postgresql.util.PSQLException
+                    Caused by: java.net.ConnectException
+
+uk.org.tombolo.core.AttributeTest > testWriteJSON FAILED
+    java.util.ServiceConfigurationError
+        Caused by: org.hibernate.service.spi.ServiceException
+            Caused by: org.hibernate.exception.JDBCConnectionException
+                Caused by: org.postgresql.util.PSQLException
+                    Caused by: java.net.ConnectException
+
+uk.org.tombolo.core.DatasourceTest > testWriteJSON FAILED
+    java.util.ServiceConfigurationError
+        Caused by: org.hibernate.service.spi.ServiceException
+            Caused by: org.hibernate.exception.JDBCConnectionException
+                Caused by: org.postgresql.util.PSQLException
+                    Caused by: java.net.ConnectException
+```
+
+The former error log is launched if the server is not running and to solve it you need to run the command.
 
-# Create DB tables and load initial fixtures
-psql -d tombolo_test -U tombolo_test < src/main/resources/sql/create_database.sql
+```bash
+pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log start
+```
+
+In case you see this other error instead, it means that you did not rename the settings files successfully.
+
+```bash
+FAILURE: Build failed with an exception.
+
+* Where:
+Build file '/TomboloDigitalConnector/build.gradle' line: 159
+
+* What went wrong:
+Execution failed for task ':test'.
+> Test environment not configured. See the README.
```

Comment:
the user is in readme

Reply:
This is a log printed to the terminal in case of error. I just wanted to reproduce the error as shown when it occurs.